sdks/python/apache_beam/yaml/examples/test/map_to_fields_callable_test.py

```diff
@@ -0,0 +1,36 @@
+# coding=utf-8
```

Comment:
Having a separate test file for (nearly) every example seems overkill. Perhaps we could consolidate?